packages/frontend/src/app.html

```diff
@@ -6,6 +6,11 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		%sveltekit.head%
 	</head>
+	<style>
+		body {
```

Comment:
I think we can add a basic CSS reset here for the base of the app. What do you think? I got the below from chatGPT, but, we can tweak to our taste:

```css
/* Basic Reset */
*,
*::before,
*::after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

body {
  line-height: 1.5;
  font-family: Arial, sans-serif;
}

img,
picture,
video,
canvas {
  display: block;
  max-width: 100%;
}

input,
button,
textarea,
select {
  font: inherit;
}

a {
  text-decoration: none;
  color: inherit;
}
```

Reply:
Yh I agree with this, I thought of  adding some of these but I wasn't sure of the file they'd be added

Reply:
> Yh I agree with this, I thought of adding some of these but I wasn't sure of the file they'd be added

This should help: https://stackblitz.com/edit/sveltejs-kit-template-default-stn1sy?file=src%2Fstyles%2Fvariables.scss

Reply:
I've added this to a style folder